molecule/default/molecule-with-systemd.yml

```diff
@@ -10,15 +10,17 @@ dependency:
 driver:
   name: docker
 platforms:
-  - cgroupns_mode: host
-    command: /lib/systemd/systemd
-    image: geerlingguy/docker-amazonlinux2023-ansible:latest
-    name: amazonlinux2023-systemd
-    platform: amd64
-    pre_build_image: yes
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+  #  Amazon Linux 2023 does not yet support Docker Compose. See the package
```

Comment:
With https://github.com/cisagov/ansible-role-docker/pull/70 this _should_ be supported. Would you please enable this platform and test?

Reply:
It fails during the `Upgrade system packages` Task with _"No file was found while using first_found"_

Reply:
@king-alexander Please update your local copy of the dependencies with `ansible-galaxy install --force --role-file molecule/default/requirements.yml` and let me know if it still fails. It works for me locally so I just want to confirm.

Reply:
@mcdonnnj Yes it still fails on my end.

Reply:
@king-alexander I think you have an old Docker image pulled or an out-of-date Ansible role because this appears to work fine per [this Actions run](https://github.com/cisagov/ansible-role-admiral/actions/runs/6227377482/job/16901890566). You might try running the `update_molecule_images.sh` script to make sure you have the latest versions of the Docker images in the molecule configuration.

Reply:
We are good to go! I rebuilt my environment and updated the Docker images with the script. That seemed to do the trick.